src/Datadog.Trace.Ci.Shared/Datadog.Trace.Ci.Shared.shproj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Actually, I'm having second thoughts on SharedProject since this is not an SDK style project. Can we instead make this a class library? What downsides are there with that approach?

Reply:
My motivation is that this code can be built from source on Linux / macOS as well. If that's the case today with this SharedProject instead of a class library, then this change is fine with me 👍

Reply:
I get the feeling that this file is for compatibility for legacy projects and the `.projitems` file can be included in SDK-style projects. But I'm just guessing 😄

Reply:
I just realized that the integration tests were building and running so it was obviously working 😅

Reply:
I’m 97% sure that will work in macos/linux, the reason is that this was very popular in xamarin projects using the xamarin ide now visual studio for mac. I’ll test all platforms before merging. 😊